golemfactory/clay
# Aio monitor

fixes: #3583

# How to test
This PR introduces changes in internal mechanism of reporting status to monitor. General communication between golem and stats.golem.network should be checked.

Also some care should be taken to check overall golem stability. Possibly issues with golem shutdown (does it take significantly more time than standard develop?). Lastly long running test should be performed to confirm wether golem takes significantly more resource than pure develop version.
